eng/pipelines/templates/jobs/archetype-sdk-tests.yml

```diff
@@ -55,10 +76,18 @@ jobs:
           OSName: $(OSName)
 
       - task: UseDotNet@2
-        displayName: "Use .NET Core runtime $(DotNetCoreRuntimeVersion)"
+        condition: eq(variables['TestTargetFramework'], 'netcoreapp2.1')
```

Comment:
We should be consistent and just set DotNetCoreRuntimeVersion in the matrix entry like you did in the other template instead of having this conditioned step